src/tilde_timing_monitor/src/tilde_timing_monitor_debug.cpp

```diff
@@ -0,0 +1,299 @@
+// Copyright 2022 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "tilde_timing_monitor/tilde_timing_monitor_debug.hpp"
+
+#include "tilde_timing_monitor/tilde_timing_monitor_core.hpp"
+
+#define FMT_HEADER_ONLY
+#include <fmt/format.h>
+
+namespace tilde_timing_monitor
+{
+// const
+static const char * tm_command_topic = "tilde_timing_monitor_command";
```

Comment:
```suggestion
static const char * const tm_command_topic = "tilde_timing_monitor_command";
```

あるいは constexpr std::string_view など。

Reply:
Is it ok to do "static constexpr char xx[] = "yy";" ?